x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_data_streams_stats/index.ts

```diff
@@ -8,25 +8,42 @@
 import type { ElasticsearchClient } from '@kbn/core/server';
 import { DataStreamType } from '../../../../common/types';
 import { dataStreamService } from '../../../services';
+import { indexStatsService } from '../../../services';
 
-export async function getDataStreamsStats(options: {
+export async function getDataStreamsStats({
+  esClient,
+  type,
+  datasetQuery,
+  sizeStatsAvailable = true,
+}: {
   esClient: ElasticsearchClient;
   type?: DataStreamType;
```

Comment:
Why not having a default like in other functions in routes? I saw we are using `logs` by default

Reply:
That we might beed to change as well